File: src/main/java/ai/aimachineserver/AimachineServerApplication.java
package ai.aimachineserver;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Bean;
import org.springframework.web.client.RestTemplate;

@SpringBootApplication
public class AimachineServerApplication {

    public static void main(String[] args) {
        SpringApplication.run(AimachineServerApplication.class, args);
    }

    @Bean
    RestTemplate restTemplate() {
        return new RestTemplate();
    }
}

File: src/main/java/ai/aimachineserver/AimachineServerApplication.kt
package ai.aimachineserver

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.Bean
import org.springframework.web.client.RestTemplate

@SpringBootApplication
class AimachineServerApplication {

    @Bean
    fun restTemplate() = RestTemplate()
}

fun main(args: Array<String>) {
    runApplication<AimachineServerApplication>(*args)
}
